api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -222,6 +250,63 @@ public Credential getCredentialFromActivityResult(@Nullable Intent resultData) {
         return responders;
     }
 
+    @Nullable
+    private ComponentName getPreferredProvider(@NonNull List<ComponentName> providers) {
```

Comment:
Is it your intention to skip the picker when there are untrusted providers and one preferred provider?